ModelLibrary/Process/BankStatementPayment.cs

```diff
@@ -96,7 +97,7 @@ private String CreatePayment(X_I_BankStatement ibs)
             MPayment payment = CreatePayment(ibs.GetC_Invoice_ID(), ibs.GetC_BPartner_ID(),
                 ibs.GetC_Currency_ID(), ibs.GetStmtAmt(), ibs.GetTrxAmt(),
                 ibs.GetC_BankAccount_ID(),Utility.Util.GetValueOfDateTime(ibs.GetStatementLineDate() == null ? ibs.GetStatementDate() : ibs.GetStatementLineDate()),
-                Utility.Util.GetValueOfDateTime(ibs.GetDateAcct()), ibs.GetDescription(), ibs.GetAD_Org_ID());
+                Utility.Util.GetValueOfDateTime(ibs.GetDateAcct()), ibs.GetDescription(), ibs.GetAD_Org_ID(), 0, 0); //Used Zero's as parameters to Avoid throw Error
             if (payment == null)
             {
                 throw new SystemException("Could not create Payment");
```

Comment:
need actual message, why payment not created

Reply:
Here not need Change anything sir, we are not using this Method as you said, for present requirement using CreatePayment(MBankStatementLine bsl) not CreatePayment(X_I_BankStatement ibs)!